packages/fxa-react/components/LogoLockup/en.ftl

```diff
@@ -0,0 +1,3 @@
+## FxA React - Strings shared between multiple FxA products for logo lockup
+
+app-logo-alt = { -brand-firefox } logo
```

Comment:
`-brand-firefox` is the term ID used in fxa-settings, but this will not work if `LogoLockup` is imported into fxa-payments-server (not currently used there, so okay at the moment) because fxa-payments-server has `-brand-name-firefox` for this term.

This will be resolved in a future ticket where fxa-payments-server will be refactored to use `fxa-shared`'s `branding.ftl` file